app/components/HomeController.react.js

```diff
@@ -30,7 +30,7 @@ export default class HomeController extends React.Component {
   render() {
     return(
       <div className='home'>
-        {routerUtils.childrenWithProps(this.props.children)}
+        <SignIn />
```

Comment:
@andela-aadepoju Actually, I don't want us to use it this way. This routerUtils is necessary to be here.